followthemoney/proxy.py

```diff
@@ -205,26 +205,31 @@ def unsafe_add(
         cleaned: bool = False,
         fuzzy: bool = False,
         format: Optional[str] = None,
-    ) -> None:
+    ) -> Optional[str]:
         """A version of `add()` to be used only in type-checking code. This accepts
         only a single value, and performs input cleaning on the premise that the
-        value is already valid unicode."""
+        value is already valid unicode. Returns the value that has been added."""
```

Comment:
This doesn’t feel like a super clean API decision, but as this method is mostly used internally, that’s probably fine and I couldn’t really come up with a better alternative.

Reply:
I don't think it's a bad promise to make... We just want `.unsafe_add` to be super type safe - because all of the rest of the proxy is a bit fuzzy about that.